src/platforms/php/guides/symfony/performance/pm-integrations.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Performance Integrations
+description: "Learn about the the integrations Sentry provides for performance monitoring and how to configure them."
+sidebar_order: 500
+---
+
+The Symfony SDK offers integrations that support distributed tracing, including database queries, Twig template rendering, and cache pools. These integrations are enabled by default.
```

Comment:
As already asked above, can we be more generic on this so that we won't have to remember that this note needs to be updated all the times we add a new integration?

Reply:
Yes, I agree, it makes the content more scalable.